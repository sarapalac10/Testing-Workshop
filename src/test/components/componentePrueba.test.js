import ComponentePrueba from "./componentePrueba"
import { shallow } from 'enzyme'


describe('Prueba en Componente Prueba', () => {
    test('Validar render en componente de prueba', () => {
        //Mesa de trabajo
        let saludos = 'Saludos :D '
        const wrapper = shallow(<ComponentePrueba saludos={saludos} />)
        expect(wrapper).toMatchSnapshot()
    })
})

