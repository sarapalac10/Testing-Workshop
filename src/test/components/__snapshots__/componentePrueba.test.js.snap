// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en Componente Prueba Validar render en componente de prueba 1`] = `
<div>
  <p>
    'hola' + 
    Saludos :D 
  </p>
</div>
`;
